arkouda/pdarrayclass.py

```diff
@@ -764,7 +764,7 @@ def __invert__(self):
             return self._binop(~0, "^")
         if self.dtype == akuint64:
```

Comment:
it's not exactly clear to me when we should use `akuint64` vs `akdtype("int64")`

Reply:
Technically, pdarray.dtype is a dtype, and therefore it is equal to `akdtype("int64")`.  However, it will be satisfy equality with `'int64' `and `int64`, which is good for backward compatibility.

```
In [11]: x = ak.arange(10)

In [12]: x.dtype == 'int64'
Out[12]: True

In [13]: x.dtype == ak.dtypes.dtype('int64')
Out[13]: True

In [14]: x.dtype == ak.dtypes.int64
Out[14]: True

In [15]: type(x.dtype)
Out[15]: numpy.dtypes.Int64DType

In [16]: type(ak.dtypes.dtype('int64'))
Out[16]: numpy.dtypes.Int64DType

In [17]: type( ak.dtypes.int64)
Out[17]: type
```